### ISSUE TITLE
[4.0] Login page naming of hidden text

### ISSUE THREAD

**Issue by unknown-sudo** [joomla/joomla-cms#26316]
There are two icons on the login page that are not immediately logical even for sighted people. One is the symbol to get into the frontend. The title of this symbol is "Preview". In my opinion this is not a sufficient term in terms of accessibility. Apart from the fact that the semantic of this element is rather bent and should be done differently (see my issue on usage of icon font vs. SVG Sprites), it should be replaced for screenreaders at least with "Preview Frontend", "Open Website in a new window "or similar. Here it wouldn't bother the layout if sighted people knew exactly what this symbol does.
The other problem can be seen in the button with the eye. The button simply says "Show" - here it should be clearly communicated that by clicking on this button the password is displayed. For example by "Show Password".

**Comment by brianteeman** [comment]
Absolutely correct about the text - please submit a pull request

**Comment by unknown-sudo** [comment]
Allright, which words do we use for the frontend? 
- (Show || Preview)  + (frontend || site || website)
- Show Password / Hide Password ?

Another problem occurs with the title of the preview button, as said, the semantics is not nice and as you mentioned in the other issue, the title is not sufficient for accessibility, so this icon is probably not accessible at all?

**Comment by brianteeman** [comment]
> (Show || Preview) + (frontend || site || website)

Show Frontend

> Show Password / Hide Password ?

Yes

**Comment by Quy** [comment]
See PR #27425 for Show/Hide  Password.

**Comment by Quy** [comment]
See PR #27461 for Show Frontend
